SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,111 @@
+<%(TOC)%>
+# Game client interest management
+
+The GDK uses [interest]({{urlRoot}}/content/glossary#interest) to control the information that [game clients]({{urlRoot}}/content/glossary#client-workers) receive about the world around them.
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
+
+You can define your game client’s interest in three ways, which you can use alongside each other:
+
+* **`NetCullDistanceSquared`**<br>
+A property that you define on an Actor’s class. For any Actor of that class or a derived class, game clients whose players are within the distance you specify receive updates about the Actor.<br>
```

Comment:
When do they receive the updates? Is it automatic? Per tick? etc etc.

Reply:
I have no idea... 😳  @Vatyx is this something we should clarify?

Reply:
This is based on the frequency value you set (which is in the same section in blueprints) which people already know about. I don't think we should clarify.